hibernate-core/src/test/java/org/hibernate/test/hql/FetchNonRootRelativeElementCollectionAndAssociationTest.java

```diff
@@ -0,0 +1,71 @@
+package org.hibernate.test.hql;
+
+import org.hibernate.jpa.test.BaseEntityManagerFunctionalTestCase;
+import org.hibernate.testing.TestForIssue;
+import org.junit.Test;
+
+import javax.persistence.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static javax.persistence.CascadeType.ALL;
+import static org.hibernate.testing.transaction.TransactionUtil.doInJPA;
+
+/**
+ * @author Moritz Becker (moritz.becker@ordami.com)
+ * @date 12/01/2019
+ * @company ordami GmbH
+ */
+@TestForIssue(jiraKey = "HHH-13201")
+public class FetchNonRootRelativeElementCollectionAndAssociationTest extends BaseEntityManagerFunctionalTestCase {
```

Comment:
Threre are quite a lot of formatting issues in this class.

Could you set up your IDE with our styles: https://github.com/hibernate/hibernate-ide-codestyles and reformat it properly?

(Checkstyle does not enforce styles in tests but let's try to be consistent.)

Thanks!